README.md

```diff
@@ -8,42 +8,17 @@ implementation from Agenzia per l'Italia Digitale.
 
 ## Private key, CSR and Self-signed certificate for public sector (with Docker)
 
-1.  Build the Docker image
+1.  Copy the file `gencert-env.public.example.sh` into `gencert-env.sh` and edit it according to your
+    needs, following the rules in the bottom section **Configuration parameters**
```

Comment:
For each documentation step, I would keep the following structure:

* Description of the step (for humans)
* Relative command line

So, in this case, it could be

1. Create your own copy of `gencert-env.sh` and customize it
   ```
   $ cp gencert-env-public.example.sh gencert-env.sh
   $ editor gencert-env.sh
   ```

Reply:
done! 👍🏻
@psmiraglia please resolve the conversation if the edits are ok!

Reply:
Ok from you side. I think I'll push something about the README file.